core/ops_builtin.rs

```diff
@@ -24,29 +21,44 @@ pub(crate) fn init_builtins() -> Extension {
       "02_error.js",
     ))
     .ops(vec![
-      ("op_close", op_sync(op_close)),
-      ("op_try_close", op_sync(op_try_close)),
-      ("op_print", op_sync(op_print)),
-      ("op_resources", op_sync(op_resources)),
-      ("op_wasm_streaming_feed", op_sync(op_wasm_streaming_feed)),
-      ("op_wasm_streaming_abort", op_sync(op_wasm_streaming_abort)),
-      (
-        "op_wasm_streaming_set_url",
-        op_sync(op_wasm_streaming_set_url),
-      ),
-      ("op_metrics", op_sync(op_metrics)),
-      ("op_void_sync", void_op_sync()),
-      ("op_void_async", void_op_async()),
-      // TODO(@AaronO): track IO metrics for builtin streams
-      ("op_read", op_async(op_read)),
-      ("op_write", op_async(op_write)),
-      ("op_shutdown", op_async(op_shutdown)),
+      op_close::decl(),
+      op_try_close::decl(),
+      op_print::decl(),
+      op_resources::decl(),
+      op_wasm_streaming_feed::decl(),
+      op_wasm_streaming_abort::decl(),
+      op_wasm_streaming_set_url::decl(),
+      op_void_sync::decl(),
+      op_void_async::decl(),
+      // // TODO(@AaronO): track IO metrics for builtin streams
+      op_read::decl(),
+      op_write::decl(),
+      op_shutdown::decl(),
+      op_metrics::decl(),
     ])
     .build()
 }
 
+mod deno_core {
+  pub use crate::*;
+}
+
+#[op]
+pub fn void_op_sync(_: &mut OpState, _: (), _: ()) -> Result<(), Error> {
+  Ok(())
+}
+
+pub async fn void_op_async(
+  _state: Rc<RefCell<OpState>>,
```

Comment:
common thing across the codebase, only one single place where just `_` is used for it. just gives more context; just nicer to read

Reply:
At some level I would've presumed that Rust might do more code elimination with `_` vs `_state` but on the other hand, probably not since at the end of the day the compiler will know exactly and equally well if `_state` is unused or not.